app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -394,6 +394,9 @@
           grid-gap: 0 1em;
           a {
             @include ellipsis;
+            > div {
```

Comment:
Minor, but could we add a class for this to the template? The css here is already a bit iffy (`ul a > div`)